docs/core/tools/dotnet-publish.md

```diff
@@ -88,13 +88,17 @@ Doesn't perform an implicit restore when running the command.
 Specifies the path for the output directory. If not specified, it defaults to *./bin/[configuration]/[framework]/* for a framework-dependent deployment or *./bin/[configuration]/[framework]/[runtime]* for a self-contained deployment.
 If a relative path is provided, the output directory generated is relative to the project file location, not to the current working directory.
 
+`-r|--runtime <RUNTIME_IDENTIFIER>`
+
+Publishes the application for a given runtime. This is used when creating a [self-contained deployment (SCD)](../deploying/index.md#self-contained-deployments-scd). For a list of Runtime Identifiers (RIDs), see the [RID catalog](../rid-catalog.md). Default is to publish a [framework-dependent deployment (FDD)](../deploying/index.md#framework-dependent-deployments-fdd).
```

Comment:
nit: Default --> The default

Reply:
Thanks @rpetrusha. I actually added "The default behavior"